docs/user/openstack/customization.md

```diff
@@ -1,29 +1,55 @@
 # OpenStack Platform Customization
 
-In the OpenShift Installer `install-config.yaml` you can set the following options regarding the OpenStack platform:
+Beyond the [platform-agnostic `install-config.yaml` properties](../customization.md#platform-customization), the installer supports additional, OpenStack-specific properties.
 
-- `machines.platform.openstack.region` - The OpenStack region where the cluster will get created
-- `machines.platform.openstack.cloud` - Name of the OpenStack cloud to use from clouds.yaml
-- `machines.platform.openstack.externalNetwork` - The OpenStack external network name to be used for installation
-- `machines.platform.openstack.computeFlavor` - The OpenStack compute flavor to use for master servers
-- `machines.platform.openstack.lbFloatingIP` - Existing Floating IP to associate with API loadbalancer
-- `machines.platform.openstack.trunkSupport` - Whether OpenStack ports can be trunked. True or False
-- `machines.platform.openstack.octaviaSupport` - Whether OpenStack supports Octavia. True of False
-- `machines.platform.openstack.defaultMachinePlatform` - (optional) The default configuration used when installing on OpenStack for machine pools
+## Cluster-scoped properties
 
-For more technical definitions, see the [go docs](https://godoc.org/github.com/openshift/installer/pkg/types/openstack#Platform).
+* `cloud` (required string): The name of the OpenStack cloud to use from `clouds.yaml`.
+* `computeFlavor` (required? string): The OpenStack compute flavor to use for control-plane machines (FIXME: how is this different from `controlPlane.platform.openstack.type`?).
+* `externalNetwork` (required string): The OpenStack external network name to be used for installation.
+* `lbFloatingIP` (required? string): Existing Floating IP to associate with the API load balancer.
+* `octaviaSupport` (optional string): Whether OpenStack supports Octavia (`1` for true or `0` for false)
+* `region` (required string): The OpenStack region where the cluster will be created.
```

Comment:
we don't use this field, so it can be removed from the document

Reply:
> we don't use this field, so it can be removed from the document

I'll defer that change to #2332, so the removal happens in one atomic pivot.